Makefile

```diff
@@ -74,6 +74,23 @@ ifeq (, $(SHELLCHECK_COMMAND))
 endif
 SHELL_FILES := $(shell find $(MAKEFILE_DIR) -type f -name '*.sh')
 
+.PHONY: lint
+transform-al2-to-al2023-config:
+	cat $(PACKER_TEMPLATE_FILE) | \
+	jq '._comment = "All template variables are enumerated here; and most variables have a default value defined in eks-worker-al2023-variables.json"' | \
+	jq '.variables.temporary_key_pair_type = "ed25519"' \
+	> $(subst al2,al2023,$(PACKER_TEMPLATE_FILE))
+
+	# TODO: switch to minimal "al2023-ami-minimal-2023.*-kernel-6.1-x86_64"
+	cat $(PACKER_DEFAULT_VARIABLE_FILE) | \
+	jq '.ami_component_description = "(k8s: {{ user `kubernetes_version` }}, containerd: {{ user `containerd_version` }})"' | \
+	jq '.ami_description = "EKS-optimized Kubernetes node based on Amazon Linux 2023"' | \
+	jq '.containerd_version = "*" | .runc_version = "*" | .docker_version = "" ' | \
+	jq '.source_ami_filter_name = "al2023-ami-2023.*-kernel-6.1-x86_64"' | \
+	jq '.launch_block_device_mappings_volume_size = "8"' | \
```

Comment:
Does `packer` create a fresh file system when you do this? Or is this really just setting the launch volume size?

Reply:
I ask as I've seen customer EKS worker nodes with kind of horrific file system usage patterns that would have been somewhat alleviated by having all the data on a fresh (somewhat optimized) `mkfs` of a large (e.g. 1TB) EBS volume rather than growing from the 2GB or 8GB file system that we started with.

Perhaps there's an opportunity to move to having the general advice being to have a separate EBS volume for all the k8s things?

Reply:
if we change the volume size, we'll want to pick one that works for cached AMIs

> Perhaps there's an opportunity to move to having the general advice being to have a separate EBS volume for all the k8s things?

Yes - containerd and kubelet should have their own volume

Reply:
ignore this `launch_block_device_mappings_volume_size`, am rolling this back

Reply:
@bryantbiggs not in scope for this PR :) we can open another issue and track if you all think that's a good idea.